Week 4/Word Break/TEBOW.txt

```diff
@@ -0,0 +1,41 @@
+Word Break:
```

Comment:
Thanks for uploading your TEBOW IT! You did an especially good job of picking small test cases that are easy to walkthrough and taking care to consider edge cases.